contracts/utils/Zapper.sol

```diff
@@ -0,0 +1,230 @@
+                amount1Min: 0,
+                recipient: msg.sender,
+                deadline: block.timestamp + 1
+            })
+        );
+        emit MintedNFT(tokenId);
+    }
+
+    ///@notice burns a uni NFT with a single output token
+    ///@param tokenId id of the NFT to burn
+    ///@param tokenOut address of output token
+    function zapOut(uint256 tokenId, address tokenOut) public {
+        (address token0, address token1) = NFTHelper._getTokenAddress(tokenId, nonfungiblePositionManager);
+
+        (, , , , , , , uint128 liquidity, , , , ) = nonfungiblePositionManager.positions(tokenId);
+
+        nonfungiblePositionManager.decreaseLiquidity(
+            INonfungiblePositionManager.DecreaseLiquidityParams({
+                tokenId: tokenId,
+                liquidity: liquidity,
+                amount0Min: 0,
+                amount1Min: 0,
+                deadline: block.timestamp + 1
+            })
+        );
+
+        (uint256 amount0, uint256 amount1) = nonfungiblePositionManager.collect(
+            INonfungiblePositionManager.CollectParams({
+                tokenId: tokenId,
+                recipient: address(this),
+                amount0Max: 2**128 - 1,
+                amount1Max: 2**128 - 1
+            })
+        );
+
+        nonfungiblePositionManager.burn(tokenId);
+
+        if (tokenOut != token0) {
+            ERC20Helper._approveToken(token0, address(swapRouter), amount0);
+
+            amount0 = swapRouter.exactInputSingle(
+                ISwapRouter.ExactInputSingleParams({
+                    tokenIn: token0,
+                    tokenOut: tokenOut,
+                    fee: _findBestFee(token0, tokenOut),
+                    recipient: address(this),
+                    deadline: block.timestamp + 1,
+                    amountIn: amount0,
+                    amountOutMinimum: 1,
+                    sqrtPriceLimitX96: 0
+                })
+            );
+        }
+
+        if (tokenOut != token1) {
+            ERC20Helper._approveToken(token1, address(swapRouter), amount1);
+
+            amount1 = swapRouter.exactInputSingle(
+                ISwapRouter.ExactInputSingleParams({
+                    tokenIn: token1,
+                    tokenOut: tokenOut,
+                    fee: _findBestFee(tokenOut, token1),
+                    recipient: address(this),
+                    deadline: block.timestamp + 1,
+                    amountIn: amount1,
+                    amountOutMinimum: 1,
+                    sqrtPriceLimitX96: 0
+                })
+            );
+        }
+
+        ERC20Helper._approveToken(tokenOut, address(this), amount0 + amount1);
+        ERC20Helper._withdrawTokens(tokenOut, msg.sender, amount0 + amount1);
+    }
+
+    ///@notice orders token addresses
+    ///@param token0 address of first token
+    ///@param token1 address of second token
+    ///@return address ordered token address
+    ///@return address ordered token address
```

Comment:
Duplicated line

Reply:
The function has two outputs, it's correct that there are two lines, but it's true that I should diversify the comments